app/Repositories/Admin/Project/ProjectRepository.php

```diff
@@ -193,6 +197,21 @@ public function togglePublicStatus($project): string
         } else {
             $this->indexProjects([$project->id], 'is_public');
         }
-        return 'Public Status toggled Successfully!';
+        return 'Public status toggled successfully!';
+    }
+
+    /**
+     * @param $projects
+     * @param $flag
+     * @return string
+     * @throws GeneralException
+     */
+    public function toggleStarter($projects, $flag): string
+    {
+        if (empty($projects)) {
+            throw new GeneralException('Choose at-least one project.');
+        }
+        $this->model->whereIn('id', $projects)->update(['starter_project' => (bool)$flag]);
+        return 'Projects Starter status updated successfully!';
```

Comment:
Sorry, one more. I think it's more meaningful to use "Starter Projects status".

Reply:
Sure, Just a sec.

Reply:
Done